hera_cal/abscal_funcs.py

```diff
@@ -1055,56 +1063,83 @@ def interp2d_vis(model, model_lsts, model_freqs, data_lsts, data_freqs,
 
     data_freqs : 1D array of the data freq axis, dtype=float, shape=(Nfreqs,)
 
-    kind : kind of interpolation method, type=str, options=['linear', 'cubic', ...]
-        see scipy.interpolate.interp2d for details
+    flags : type=DataContainer, dictionary containing model flags. Can also contain model wgts and
```

Comment:
Explain what happens if you just use the default (None)

Reply:
Also clarify that by "can also contain model wgts" you mean if it's floats rather than bools.